File: win-prog/ProgWinVB/Edit List and Spin/NotepadCloneNoMenu/NotepadCloneNoMenu.vb
'---------------------------------------------------
' NotepadCloneNoMenu.vb (c) 2002 by Charles Petzold
'---------------------------------------------------
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Class NotepadCloneNoMenu
    Inherits Form

    Protected txtbox As TextBox

    Shared Sub Main()
        Application.Run(New NotepadCloneNoMenu())
    End Sub

    Sub New()
        Text = "Notepad Clone No Menu"

        txtbox = New TextBox()
        txtbox.Parent = Me
        txtbox.Dock = DockStyle.Fill
        txtbox.BorderStyle = BorderStyle.None
        txtbox.Multiline = True
        txtbox.ScrollBars = ScrollBars.Both
        txtbox.AcceptsTab = True
    End Sub
End Class

File: win-prog/ProgWinCS/Edit List and Spin/NotepadCloneNoMenu/NotepadCloneNoMenu.cs
//-------------------------------------------------
// NotepadCloneNoMenu.cs © 2001 by Charles Petzold
//-------------------------------------------------
using System;
using System.Drawing;
using System.Windows.Forms;

class NotepadCloneNoMenu: Form
{
     protected TextBox txtbox;

     public static void Main()
     {
          Application.Run(new NotepadCloneNoMenu());
     }
     public NotepadCloneNoMenu()
     {
          Text = "Notepad Clone No Menu";

          txtbox             = new TextBox();
          txtbox.Parent      = this;
          txtbox.Dock        = DockStyle.Fill;
          txtbox.BorderStyle = BorderStyle.None;
          txtbox.Multiline   = true;
          txtbox.ScrollBars  = ScrollBars.Both;
          txtbox.AcceptsTab  = true;
     }
}          
